File: src/main/java/com/senderman/lastkatkabot/TempObjects/BnCPlayer.java
package com.senderman.lastkatkabot.TempObjects;

public class BnCPlayer extends TgUser {

    private final int score;

    public BnCPlayer(int id, String name, int score) {
        super(id, name);
        this.score = score;
    }

    public int getScore() {
        return score;
    }
}

File: src/main/java/com/senderman/lastkatkabot/tempobjects/BnCPlayer.kt
package com.senderman.lastkatkabot.tempobjects

import com.senderman.TgUser

class BnCPlayer(id: Int, name: String, val score: Int) : Comparable<BnCPlayer>, TgUser(id, name) {
    override fun compareTo(other: BnCPlayer): Int = this.score - other.score
}